data/json/bionics.json

```diff
@@ -589,7 +589,7 @@
       [ "hand_r", 1 ],
       [ "mouth", 1 ]
     ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ],
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "IRRITANT_IMMUNE" ],
```

Comment:
Under the skin won’t be irritant immune. I’m not even sure it should be shockproof

Reply:
I can't remember what I thought process behind that was. But I believe the `BIONIC_SHOCKPROOF` flag just means if you get electrocuted it won't get messed up.